### PR TITLE
Add prioritization guide

### PR DESCRIPTION
**Pull request by justinretzolk**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Output from acceptance testing: N/a, docs

### CODE REVIEW

File: docs/contributing/faq.md [hunk 1]
```diff
@@ -26,9 +27,7 @@ We have a large backlog of pull requests to get through and the team are moving
...
 
-Once we have prioritized your contribution for review, we will let you know when to expect an engineer to get in touch. If changes are required, we will ask in the pull request. If you are unable to find the time, just let us know, and we will make the necessary changes required in order to merge.
-
-We publish a [road map](../../ROADMAP.md) every quarter which describes major themes or specific product areas of focus.
+We publish a [road map](../../ROADMAP.md) every quarter which describes major themes or specific product areas of focus. What is excluded from the public roadmap is work performed under NDA with AWS on new services, and any ad-hoc work we pick up during the quarter. This ad-hoc work can be responding to bugs, gardening day activity, customer prioritization, and technical debt items.
```
Comment (ewbankkit) [inline docs/contributing/faq.md:30]: `road map` or `roadmap`? Both are used in this paragraph.

Comment (justinretzolk) [reply docs/contributing/faq.md:30]: Elsewhere, it seems like we usually use `roadmap`, so I'll get that corrected 🙂

**Review by ewbankkit — APPROVED** [review]
LGTM :rocket:.

File: docs/contributing/core-services.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Terraform AWS Provider Core Services
 
-Core Services are AWS services we have identified as critical for a large majority of our users. Our goal is to continually increase the depth of coverage for these services. We will work to prioritize features and enhancements to these services in each weekly release, even if they are not necessarily highlighted in our quarterly road map.
+Core Services are AWS services we have identified as critical for a large majority of our users. Our goal is to continually increase the depth of coverage for these services. We will work to prioritize features and enhancements to these services in each weekly release, even if they are not necessarily highlighted in our quarterly roadmap.
```
Comment (AdamTylerLynch) [inline docs/contributing/core-services.md:3]: Can we define what those services are explicitly?

**Review by AdamTylerLynch — COMMENTED** [review]
Adding an explicit list of "core services" would be helpful.

File: docs/contributing/core-services.md (hunk 2 above)
Comment (justinretzolk) [reply docs/contributing/core-services.md:3]: Great suggestion! I'll push a quick edit to link out to [core services](https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/contributing/core-services.md)

Comment (justinretzolk) [reply docs/contributing/core-services.md:3]: Oh, I think I may have misunderstood here. Where were you suggesting to add the explicit list of core services -- to the prioritization guide?

Comment (justinretzolk) [reply docs/contributing/core-services.md:3]: 🎉 Triple reply! I think I follow you now. I pushed a change to the `core-services.md` file as a part of this PR just to make our use of `roadmap` vs `road map` consistent. There's an explicit list of the core services below this edited line in the same document, which we do link out to from the prioritization guide already.

**Pull request merged**